### DISCUSSION TITLE
I got banned randomly from discord.py and i wonder tò know why

### DISCUSSION THREAD

**Discussion by vincyxiroff** [Rapptz/discord.py#9889] [Q&A]
My user id Is: 1017694304036200469

**Comment by MCausc78** [comment]
You was making a stealer/logger.
![Screenshot_20240718-153952_Discord.jpg](https://github.com/user-attachments/assets/9c716516-d4ef-4911-b96e-8f09eaee0a51)

**Reply by vincyxiroff** [reply]
I wasnt as u Need tò have One + i dont save anything tò myself if u dm me on discord i'll give u the code

**Reply by MCausc78** [reply]
> I wasnt as u Need tò have One + i dont save anything tò myself if u dm me on discord i'll give u the code 

Why? Why would I even steal Roblox accounts? This does not make sense, only does harm to my reputation, doesn't make any profit, and it is illegal. I do not want to make some malware with script kiddie. If you need money, then just apply for a job.

**Reply by vincyxiroff** [reply]
bro i do not steal the cookie he miss understand as h saw the class log() thta the code is this:
def log(text,sleep=None): 
    print(f"[{datetime.utcfromtimestamp(__()).strftime('%Y-%m-%d %H:%M:%S')}] → {text}")
    if sleep: zzz(sleep)


this is a big misunderstanding

**Reply by MCausc78** [reply]
You definitely do.

![Screenshot_20240718-210740_Discord.jpg](https://github.com/user-attachments/assets/b72a9a61-8a66-4da4-9bdb-8b659092eef4)

https://mystb.in/0caa716f97ac25ddc3

**Reply by vincyxiroff** [reply]
dm me on discord i'll prove you or make leonardo read my dm

**Reply by vincyxiroff** [reply]
read the code and see by yourself: https://github.com/vincyxiroff/Loxy-Loxy/blob/main/main.py


+ u can even see on dev.roblox.com that u can do it aslong u dont use to steal account
https://devforum.roblox.com/t/fastest-way-to-check-if-a-roblox-cookie-is-valid-or-invalid/511451/5
ts just api requests to official roblox website
so i didnt break any roblox tos 😭 🙏

**Reply by vincyxiroff** [reply]
did u read teh code in mystb.in i think u diddnt as  i dont store any cookie for myself and only the user get it

**Reply by Hypurrnating** [reply]
I know how roblox can be, so I understand your pov. 
but to many the idea that you need their cookies passing through your bot is absurd (rightfully so). but this is a problem with roblox and im not sure how far you will get convincing anyone else.

**Reply by Hypurrnating** [reply]
also im not sure if you didnt break roblox tos, surely having their cookie has to count as something ?

**Reply by vincyxiroff** [reply]
it does not break any kind of tods is just for dont manual checking trhe cookie used to make roblox bot for like auto approve member on a group

**Reply by vincyxiroff** [reply]
+ its not a public bot its just for myself so i dont se the issue

**Reply by Rapptz** [reply]
If it's for yourself only then it's better for you to not accept a cookie as a parameter and just have it in a config file.

**Comment by EvieePy** [comment]
This is not the place to enquire about your bans on discord. This is for issues and discussions about the library. Please contact a moderator on discord if you wish to appeal.

**Reply by davidstown** [reply]
I also made a discussion to appeal my Discord.py server ban, the problem is that I don't know any moderator of the server.
